README.md

```diff
@@ -189,6 +189,11 @@ cfg_condition = "mycond"
     name = "baseline-position"
     version = "3.10"
     ignore = true
+    [[object.property]]
+    name = "events"
+    # generate only `connect_property_events_notify`, without `get_property_events` and `set_property_events`
+    # supported values: "get", "set", "notify", also allowed union "get|set"
+    generate = "notify"
```

Comment:
Why not a toml list of strings?

Reply:
Then we need accept both array and single value: too much work on Toml types,
and only array IMHO redundantly.
Can change if @GuillaumeGomez prefer array too.